test/integration/base_integration_test.h

```diff
@@ -41,6 +41,14 @@
   } while (0)
 #endif
 
+#ifndef ENVOY_ADMIN_FUNCTIONALITY
+#define DISABLE_IF_ADMIN_DISABLED return
```

Comment:
nit: RETURN_IF_ADMIN_DISABLED may be more obvious at call-sites. If I read this in the middle of a function I might not know what DISABLE meant.

Also -- why not use this pattern in production code also? It's somewhat nice in a way :)

Reply:
sure makes sense.